### PR TITLE
MediaTomb

### PR DESCRIPTION
**Pull request by ohnx**
Attempted to update the MediaTomb recipe.

The build currently fails while attempting to locate inet_ntoa.

Just thought I'd leave this here, in case someone else wants to take a crack at this :)

### CODE REVIEW

File: net-misc/mediatomb/mediatomb-0.12.1.recipe [hunk 1]
```diff
@@ -1,31 +1,48 @@
...
+	"
+
+BUILD_PREREQUIRES="
+	cmd:libtoolize
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:make
+	cmd:gcc
+	sqlite
```
Comment (diversys) [inline net-misc/mediatomb/mediatomb-0.12.1.recipe]: non cmd should go to BUILD_REQUIRES()

**Comment by extrowerk** [comment]
inet_ntoa should be in /posix/arpa/inet.h
Haven't checked it, but maybe you want to add 
export LDFLAGS="-lnetwok"
somewhere in your recipe. Like before configure.

**Comment by scottmc** [comment]
yeah, what @extrowerk said, try putting -lnetwork in there and see if that helps.  If you are then still stuck let us know.

**Comment by ohnx** [comment]
OK, so I updated the `configure.ac` to search in `network` as well. Now, I'm coming across an error that `lwres/netdb.h` is missing...

**Comment by waddlesplash** [comment]
it's just `<netdb.h>` I believe.

**Comment by Begasus** [comment]
This can be closed? See https://github.com/haikuports/haikuports/pull/2447

**Pull request closed without merging**